packages/lib/src/classes/AutoCompleteHandler.ts

```diff
@@ -0,0 +1,55 @@
+import { AutocompleteInteraction } from "discord.js"
+import { BetterClient, ApplicationCommand, LogLevel } from "../index.js"
+
+export default class AutoCompleteHandler {
```

Comment:
autocomplete is one word therefore `autocompleteHandler`. You can use the replace all function for all files to fix this. ditto through this file and others.